lib/usb-hid-msc-lib.robot

```diff
@@ -33,3 +33,24 @@ Download ISO And Mount As USB
             Skip    unsupported
         END
     END
+
+Make ISO And Mount As USB
+    [Documentation]    first creates the right image using a script,
+    ...    then mounts it
+    [Arguments]    ${img_name}
+
+    ${out}=    Run    sudo ./scripts/secure-boot/generate-images/${img_name}.sh
```

Comment:
@WiktorG351 do we need to run it with sudo? Maybe there is some keyword builtin RF for such situations? https://robotframework.org/robotframework/

Reply:
@TomaszAIR unfortunately yes. Because we need sudo to `mount`/`umount` the image.

Reply:
@TomaszAIR I think it is perfectly fine that RB script just check if the images are available, and if not, just ask to run some preparation script first. Running of this preparation script, can be one of the steps in the `getting started`.

Reply:
At the end this is a way that we chose. Info added to README https://github.com/Dasharo/open-source-firmware-validation/pull/192/commits/16058f263b77da1e7919c2a1d5eb0b4018ddde46